operator/v1/types_ingress.go

```diff
@@ -678,7 +678,7 @@ type IBMLoadBalancerParameters struct {
 // OpenStackLoadBalancerParameters provides configuration settings that are
 // specific to OpenStack load balancers.
 type OpenStackLoadBalancerParameters struct {
-	// loadBalancerIP specifies the floating IP address that the load balancer will use.
+	// floatingIP specifies the IP address that the load balancer will use.
```

Comment:
another doc improvement (to be tested first) is that the FIP can or cannot exist, it doesn't matter. CPO will try to create it if this doesn't exist (if Neutron policy allows it).

https://github.com/kubernetes/cloud-provider-openstack/blob/333a126f63038d52fad1e306eff6ee4ef7f53948/pkg/openstack/loadbalancer.go#L630-L633

Reply:
I wonder if we could be more explicit about the requirements of this field:

`floatingIP` specifies a floating IP address that the load balancer will use. If specified, the floating IP must already have been created.

When not specified...

Reply:
when not specified, CPO will create it. Yeah we can be more specific, happy to do it.

Reply:
in fact we need to mention that the FIP has to be pre created, otherwise CPO complains:

```
│ I1002 00:17:16.488963       1 event.go:389] "Event occurred" object="openshift-ingress/router-default" fieldPath="" kind="Service" apiVersion="v1" type="Warning" reason="SyncLoadBalancerFailed" message="Error syncing load balancer: fai │
│ led to ensure load balancer: error creating LB floatingip: Request forbidden: [POST https://<neutron-endpoint>:13696/v2.0/floatingips], error message: {\"NeutronError\": {\"type\": \"PolicyNotAuthorized\", \"message\": \"(rule:create_floatingi │
│ p and rule:create_floatingip:floating_ip_address) is disallowed by policy\", \"detail\": \"\"}}
```